microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -1394,7 +1454,7 @@ def _check_version_by_pattern_value(
                 f"Failed to parse {library_name} version from output: {version_output}"
             )
 
-        current_version = Version(match.group(group_index))
+        current_version = Version(match.group(group_index).replace("-", "."))
```

Comment:
Add comments above to show examples, which need to replace chars.

Reply:
BTW, how about change it to LISA used version parser?

Reply:
Have added comments. 
`parse_version` and `LisaVersionInfo` don't support the four-part version like "1.2.3.4". Waagent versions have this format. So we used `Version`.